docs/src/user/installing.md

```diff
@@ -12,30 +12,39 @@ rustup target add --toolchain stable wasm32-wasi
 
 ## Installing from Git
 
-Ambient can be installed through `cargo install`, which will download and build the repository. Our current minimum supported Rust version is <!-- rust-version-begin !-->1.67.0<!-- rust-version-end !-->, as we use recently-stabilised standard library features.
+*Ambient* can be installed through `cargo install` using git.
```

Comment:
No emphasis (you know what you're building), and no "using git" (it's in the heading)